src/js/utils/responsive.js

```diff
@@ -1,25 +1,32 @@
 export const getBreakpoint = (windowWidth, theme) => {
   let result;
-  Object.keys(theme.global.breakpoints)
-    .sort((a, b) => {
+  const sortedBreakpoints = Object.keys(theme.global.breakpoints).sort(
+    (a, b) => {
       const first = theme.global.breakpoints[a];
       const second = theme.global.breakpoints[b];
       if (!first) return 1;
       if (!second) return -1;
       if (!first.value) return 1;
       if (!second.value) return -1;
       return first.value - second.value;
-    })
-    .some(name => {
-      const breakpoint = theme.global.breakpoints[name];
-      if (breakpoint) {
-        if (!breakpoint.value || breakpoint.value >= windowWidth) {
-          result = name;
-          return true;
-        }
+    },
+  );
+  const lastBreakpoint = sortedBreakpoints[sortedBreakpoints.length - 1];
+
+  sortedBreakpoints.some(name => {
+    const breakpoint = theme.global.breakpoints[name];
```

Comment:
I am not sure i understand what is going on here :/
what is lastBreakpoint and why we use this. Probably this is what lead to the snapshot changes?

I think this whole part is a bit confusing because we use array.some to not do anything here just set the result. maybe this was better as a filter? Do we need the last value?

Reply:
The condition in line 22 `breakpoint.value >= windowWidth` requests the value of the breakpoint to be larger than the windowWidth.

Let's say the last breakpoint size is `xlarge: { value: 1000 }`, and the client is using a monitor that has 2000 of windowWidth, our logic, in that case, should be to return the `xlarge` size, but instead we are returning `undefined` (due to `breakpoint.value >= windowWidth` returns `false`) that eventually default the breakpoint to be `medium`.
I think the last breakpoint should have no boundary with its value, and that is what this PR is suggesting. I'm not sure how it impacts the SSR tests and wanted your inputs.